lab-matt/test/player-controller-test.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
just drop
`/* global expect angular */`
at the top of your file and eslint will stop bugging you about those variable names